SPECS/liblognorm/liblognorm.spec

```diff
@@ -0,0 +1,179 @@
+%define htmldir %{_docdir}/liblognorm/html
+Summary:        Fast samples-based log normalization library
+Name:           liblognorm
+Version:        2.0.3
+Release:        11%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.liblognorm.com
+Source0:        https://www.liblognorm.com/files/download/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  chrpath
+BuildRequires:  libestr-devel
+BuildRequires:  libfastjson-devel
+BuildRequires:  pcre-devel
+BuildRequires:  python3-sphinx
+
+%description
+Briefly described, liblognorm is a tool to normalize log data.
+
+People who need to take a look at logs often have a common problem. Logs from
+different machines (from different vendors) usually have different formats for
+their logs. Even if it is the same type of log (e.g. from firewalls), the log
+entries are so different, that it is pretty hard to read these. This is where
+liblognorm comes into the game. With this tool you can normalize all your logs.
+All you need is liblognorm and its dependencies and a sample database that fits
+the logs you want to normalize.
+
+%package        devel
+Summary:        Development tools for programs using liblognorm library
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       json-c-devel%{?_isa}
+Requires:       libestr-devel%{?_isa}
+
+%description    devel
+The liblognorm-devel package includes header files, libraries necessary for
+developing programs which use liblognorm library.
+
+%package        doc
+Summary:        HTML documentation for liblognorm
+
+%description    doc
+This sub-package contains documentation for liblognorm in a HTML form.
+
+%package        utils
+Summary:        Lognormalizer utility for normalizing log files
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description utils
+The lognormalizer is the core of liblognorm, it is a utility for normalizing
+log files.
+
+%prep
+%autosetup
+
+%build
+%configure \
+    --enable-regexp \
+    --enable-docs \
+    --docdir=%{htmldir} \
+    --includedir=%{_includedir}/%{name}/
+
+%install
+%make_install INSTALL="install -p"
+rm -f %{buildroot}/%{_libdir}/*.{a,la}
+chrpath -d %{buildroot}%{_bindir}/lognormalizer
+chrpath -d %{buildroot}%{_libdir}/liblognorm.so
+rm %{buildroot}%{htmldir}/{objects.inv,.buildinfo}
+
+%ldconfig_scriptlets
+
+%files
+%license COPYING
+%doc AUTHORS ChangeLog README
+%exclude %{htmldir}
+%{_libdir}/lib*.so.*
+
+%files devel
+%{_libdir}/lib*.so
+%{_includedir}/%{name}/*.h
+%{_libdir}/pkgconfig/*.pc
+
+%files doc
+%doc %{htmldir}
+
+%files utils
+%{_bindir}/lognormalizer
+
+
+%changelog
+* Mon Jul 20 2021 Thomas Crain <thcrain@microsoft.com> - 2.0.3-11
+- License verified
+- Spec linted
+
+* Tue Jan 12 2021 Joe Schmitt <joschmit@microsoft.com> - 2.0.3-10
+- Initial CBL-Mariner import from Fedora 32
```

Comment:
Could you also add the license info like in other specs?

Reply:
Added, thank you!